buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -100,7 +100,21 @@ public void run() {
          *
          * See https://github.com/netty/netty/issues/3888.
          */
-        final int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
+        int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
+
+        // Respect MALLOC_ARENA_MAX if set.
+        final String mallocArenaMaxStr = System.getenv("MALLOC_ARENA_MAX");
+        if (mallocArenaMaxStr != null) {
+            try {
+                int value = Integer.parseInt(mallocArenaMaxStr);
+                if (value > 0) {
+                    defaultMinNumArena = Math.min(Integer.parseInt(mallocArenaMaxStr), defaultMinNumArena);
```

Comment:
I think we could use `value`.
```suggestion
                    defaultMinNumArena = Math.min(value, defaultMinNumArena);
```